DEVELOPERS.md

```diff
@@ -39,13 +39,10 @@ export TEST_ARCHIVIST="https://app.rkvst.io"
 export TEST_AUTHTOKEN_FILENAME=credentials/.auth_token
 export TEST_NAMESPACE="unique label"
 export TEST_VERBOSE=-v
-export TEST_PROOF_MECHANISM="-i SIMPLE_HASH"
 ```
 
 If TEST_VERBOSE is "-v" debugging output will appear when running the tests.
 
-TEST_PROOF_MECHANISM must be either "KHIPU" or "SIMPLE_HASH"
```

Comment:
I think this is the only change we need?  All the rest can stay as proof mechanism is not going away, just one of the options (and not the default option)